docs/vidu-docs/图片生成.md

```
@@ -0,0 +1,73 @@
+| **参数名称** | **类型** | **必填** | **参数描述** |
+| --- | --- | --- | --- |
+| model | String | 是 | 模型名称<br>可选值：`viduq2`、`viduq1`<br>`viduq2`：支持文生图、图片编辑、参考生图<br>`viduq1`：支持参考生图 |
+| images | Array\[String\] | 可选 | 图像参考<br>`viduq2`：支持输入 **0～7** 张图片<br>`viduq1`：支持输入 **1～7** 张图片<br>模型将以此参数中传入的图片中的主题为参考，生成与图片中主体一致的视频。<br>注1：支持传入图片 Base64 编码或图片URL（确保可访问)<br>注2：图片支持 **png、jpeg、jpg、webp**格式<br>注3：图片像素不能小于 128\*128，且比例需要小于1:4或者4:1 <br>注4: 且大小不超过**50M**<br>注5：请注意，http请求的post body不超过**20MB**，且编码必须包含适当的内容类型字符串，例如：<br>`data:image/png;base64,{base64_encode}` |
+| prompt | String | 是 | 文本提示词<br>视频生成的文本描述，长度不能超过 2000 个字符<br>注1：viduq2模型支持文生图，在使用viduq2模型，且没有上传任何images时，模型会使用该参数的文本内容生成图片 |
+| seed | Int | 可选 | 随机种子参数<br>当默认不传或者传0时，会使用随机数替代<br>手动设置则使用设置的种子 |
+| aspect_ratio | String | 可选 | 比例参数，不同模型支持不同的比例：<br>`viduq1`：默认值`16:9`，可选值：`16:9`、`9:16`、`1:1`、`3:4`、`4:3`<br>`viduq2`：默认值`16:9`，可选值如下：`16:9`、`9:16`、`1:1`、`3:4`、`4:3`、`21:9`、`2:3`、`3:2`<br>- `auto`：表示与首张输入图保持相同比例 |
+| resolution | String | 可选 | 分辨率参数，不同模型支持的分辨率不同：<br>`viduq1`：默认`1080p`，可选项：`1080p`<br>`viduq2`：默认`1080p`，可选项：`1080p`、`2K`、`4K` |
+| payload | String | 可选 | 透传参数<br>不做任何处理，仅数据传输<br>注：**最多 1048576个字符** |
+| callback_url | String | 可选 | Callback 协议<br>需要您在创建任务时主动设置 `callback_url`，请求方法为 POST，当视频生成任务有状态变化时，Vidu 将向此地址发送包含任务最新状态的回调请求。回调请求内容结构与[查询任务API](https://platform.vidu.cn/docs/search-task-api)的返回体一致<br>回调返回的“status”包括以下状态：<br>- `processing` 任务处理中<br>- `success` 任务完成（如发送失败，回调三次）<br>- `failed` 任务失败（如发送失败，回调三次）<br>Vidu采用回调签名算法进行认证，详情见：[回调签名算法](https://platform.vidu.cn/docs/callback-signature) |
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)